lib/suite_runner.js

```diff
@@ -49,12 +49,13 @@ function runDebugger(childProcess, debugPort, inspectorPort) {
   var inspector = childProcess.fork(
     require.resolve('node-inspector/bin/inspector'),
     ['--debug-port=' + debugPort, '--web-port=' + inspectorPort],
-    { silent: true });
+    { silent: false });
 
   inspector.on('message', function(msg) {
     if (msg.event === 'SERVER.ERROR') {
       console.error('Cannot start the debugging server:', msg.error.code);
     } else if (msg.event === 'SERVER.LISTENING') {
+      console.log('\x1b[31mWARNING: Current version (node 6.8) is not yet supported when debugging, please downgrade to Node 6.3\x1b[0m');
```

Comment:
What about node 6.6? Does that work? I think this message should say which version breaks.

Would it be possible to make this print a warning only on a version that is not supported?